### PR TITLE
update os name. reported by translator zakooch, thanks!

### PR DESCRIPTION
**Pull request by emmapeel2**
reported by translator zakooch, thanks!

### CODE REVIEW

**Comment by emmapeel2** [comment]
not sure how to go about updating the translation files with this change.

File: docs/source/tor.rst [hunk 1]
```diff
@@ -44,8 +44,8 @@ You're now running a system Tor in Windows!
 
 Open OnionShare. Click the Settings icon. Under "How should OnionShare connect to Tor?" choose "Connect using control port", and set the control port host to ``127.0.0.1`` and the port to ``9051``. Under "Tor authentication options" choose "Password" and set the password to your password, in my case ``comprised stumble rummage work avenging construct volatile``. Click the "Test Settings" button. If all goes well, you should see successfully connected to tor.
```
Comment (comradekingu) [inline docs/source/tor.rst:45]:
```suggestion
Open OnionShare. Click the "Settings" icon. Under "How should OnionShare connect to Tor?" choose "Connect using control port", and set the control port host to ``127.0.0.1`` and the port to ``9051``. Under "Tor authentication settings" choose "Password", and set your password. A good and memorable one could look something like: ``comprised stumble rummage work avenging construct volatile``. Click the "Test Connection to Tor" button. If all goes well, you should see "Connected to the Tor controller".
```
As per
https://hosted.weblate.org/translate/onionshare/translations/en/?checksum=f9d1460ed676c508 ?
https://hosted.weblate.org/translate/onionshare/translations/en/?checksum=557d0b8b5683bc87 ?
https://hosted.weblate.org/translate/onionshare/translations/en/?checksum=eb4118378382ffd0 ?

File: docs/source/tor.rst [hunk 2]
```diff
@@ -44,8 +44,8 @@ You're now running a system Tor in Windows!
...
 
-Using a system Tor in Mac OS X
-------------------------------
+Using a system Tor in macOS
+---------------------------
 
 First, install `Homebrew <http://brew.sh/>`_ if you don't already have it. Then, install Tor::
```
Comment (comradekingu) [inline docs/source/tor.rst:50]:
```suggestion
First, install `Homebrew <https://brew.sh/>`_ if you don't already have it. Then, install Tor::
```

Comment (emmapeel2) [reply docs/source/tor.rst:50]: I think this belongs to another merge request

**Review by micahflee — APPROVED** [review]

**Comment by micahflee** [comment]
> not sure how to go about updating the translation files with this change.

Ahh to do this, you just need to run the `build.sh` script in the `docs` folder: https://github.com/micahflee/onionshare/blob/develop/docs/build.sh

Also this script includes a list of locales that are built, so we should update this:

```
# Supported locales
#LOCALES="ar ca zh_CN zh_TW da nl en fr de el is ga it ja nb fa pl pt_BR pt_PT ro ru sr@latin es sv te tr uk"
LOCALES="en es"
```

**Pull request merged**